protocol.html

```diff
@@ -755,7 +755,7 @@ <h3 property="schema:name">Writing Resources</h3>
 
                   <p><span about="" id="server-protect-containment" rel="spec:requirement" resource="#server-protect-containment"><span property="spec:statement"><span rel="spec:requirementSubject" resource="spec:Server">Servers</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> NOT allow HTTP <code>POST</code>, <code>PUT</code> and <code>PATCH</code> to update a container’s containment triples; if the server receives such a request, it <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> respond with a <code>409</code> status code.</span></span> [<a href="https://github.com/solid/specification/issues/40#issuecomment-573358652" rel="cito:citesAsSourceDocument">Source</a>]</p>
 
-                  <p><span about="" id="server-protect-authoritative-resource-data" rel="spec:requirement" resource="#server-protect-authoritative-resource-data"><span property="spec:statement"><span rel="spec:requirementSubject" resource="spec:Server">Servers</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> NOT allow HTTP <code>POST</code>, <code>PUT</code> and <code>PATCH</code> to update the <a href="#authoritative-resource-data">authoritative resource data</a>; if the server receives such a request, it MUST respond with a <code>409</code> status code.</span></span> [<a href="https://github.com/solid/specification/issues/227#issuecomment-919312592" rel="cito:citesAsSourceDocument">Source</a>]</p>
+                  <p><span about="" id="server-protect-authoritative-resource-data" rel="spec:requirement" resource="#server-protect-authoritative-resource-data"><span property="spec:statement"><span rel="spec:requirementSubject" resource="spec:Server">Servers</span> <span rel="spec:requirementLevel" resource="spec:MUST-NOT">MUST NOT</span> allow HTTP <code>POST</code>, <code>PUT</code> and <code>PATCH</code> to update the <a href="#authoritative-resource-data">authoritative resource data</a>; if the server receives such a request, it MUST respond with a <code>409</code> status code.</span></span> [<a href="https://github.com/solid/specification/issues/227#issuecomment-919312592" rel="cito:citesAsSourceDocument">Source</a>]</p>
```

Comment:
I couldn't find a namespace document, so I've been using `spec:MUSTNOT`. Only one of those have been merged, so it isn't a big change, but we need to be consistent throughout, so I just want to raise it.

Reply:
It is `MUST-NOT`: https://github.com/solid/vocab/blob/specification-terms/spec.ttl